lib/zold/commands/remote.rb

```diff
@@ -112,6 +129,8 @@ def run(args = [])
       when 'update'
         update(opts)
         update(opts, false)
+      when 'select'
+        select(opts['max-nodes'])
```

Comment:
@botanicus by convention (see all other commands) we pass `opts`, not a specific option